frontend/src/components/pages/RegistrantExperience/RegistrationSteps/ReviewRegistration/ReviewInformation/EditContactCard.tsx

```diff
@@ -0,0 +1,320 @@
+import React, { useState, useMemo } from "react";
+import {
+  Box,
+  Button,
+  Divider,
+  Flex,
+  FormControl,
+  FormErrorMessage,
+  Heading,
+  Input,
+  Spacer,
+  Text,
+  Textarea,
+  Wrap,
+  WrapItem,
+} from "@chakra-ui/react";
+import {
+  PersonalInfoActions,
+  PersonalInfoReducerDispatch,
+} from "../../../../../../types/PersonalInfoTypes";
+import { EmergencyContact } from "../../../../../../types/CamperTypes";
+import {
+  checkEmail,
+  checkFirstName,
+  checkLastName,
+  checkPhoneNumber,
+  checkRelationToCamper,
+} from "../../PersonalInfo/personalInfoReducer";
+import EditFormLabel from "./EditFormLabel";
+import EditCardFooter from "./EditCardFooter";
+
+type EditContactCardProps = {
+  contact: EmergencyContact;
+  contactIndex: number;
+  dispatchPersonalInfoAction: (action: PersonalInfoReducerDispatch) => void;
+};
+
+const EditContactCard = ({
+  contact,
+  contactIndex,
+  dispatchPersonalInfoAction,
+}: EditContactCardProps): React.ReactElement => {
+  const [updateMemo, setUpdateMemo] = useState(0);
+
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  const initialContact: EmergencyContact = useMemo(() => contact, [updateMemo]);
+
+  const [editing, setEditing] = useState(false);
+
+  const [isFirstNameInvalid, setIsFirstNameInvalid] = useState<boolean>(false);
+  const [isLastNameInvalid, setIsLastNameInvalid] = useState<boolean>(false);
+  const [isEmailInvalid, setIsEmailInvalid] = useState<boolean>(false);
+  const [isPhoneNumberInvalid, setIsPhoneNumberInvalid] = useState<boolean>(
```

Comment:
same comment as before - would omit the <boolean> for consistency

Reply:
Done.